### PR TITLE
[react-native] Added missing presentationStyle to the Modal

### PR DESCRIPTION
**Pull request by alexdunne**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://facebook.github.io/react-native/docs/modal.html#presentationstyle

### CODE REVIEW

File: types/react-native/index.d.ts [hunk 1]
```diff
@@ -4399,6 +4399,10 @@ export interface ModalProperties {
      * @platform ios
      */
     onDismiss?: () => void;
+    /**
+     * The `presentationStyle` determines the style of modal to show
```
Comment (alloy) [inline types/react-native/index.d.ts]: Can you add a `@platform ios` comment like the above ones have?

**Review by alloy — CHANGES_REQUESTED** [review]
Thanks! The typing looks good, just a small comment addition request.

File: types/react-native/index.d.ts (hunk 1 above)
Comment (alexdunne) [reply types/react-native/index.d.ts]: My bad - added it now.

Comment (alloy) [reply types/react-native/index.d.ts]: Thanks!

**Review by alloy — APPROVED** [review]

**Comment by alexdunne** [comment]
Cheers 👍

**Pull request merged**